src/OpenTelemetry.Contrib.Instrumentation.Quartz/OpenTelemetry.Contrib.Instrumentation.Quartz.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>OpenTelemetry Quartz.NET Instrumentation</Description>
+    <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
+    <MinVerTagPrefix>Instrumentation.Quartz-</MinVerTagPrefix>
+    <IncludeSharedInstrumentationSource>true</IncludeSharedInstrumentationSource>
+    <TargetFrameworks>net461;net472;netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+    
+  <ItemGroup>
+    <PackageReference Include="OpenTelemetry" Version="1.0.1" />
```

Comment:
you should be able to take just a API dependency instead of SDK, once the other comments are addressed.

Reply:
updated from SDK to API